src/FamilyHubs.ServiceDirectory.Shared/Dto/Metrics/EventDto.cs

```diff
@@ -0,0 +1,10 @@
+using FamilyHubs.ServiceDirectory.Shared.Enums;
+
+namespace FamilyHubs.ServiceDirectory.Shared.Dto.Metrics;
+
+public class EventDto
+{
+    public ServiceDirectorySearchEventType? Id { get; set; }
```

Comment:
Is this an Id?

Reply:
Yes, driven by the enum `ServiceDirectorySearchEventType`. It's a lookup table for categorising metric event types. EF seeds the data with two IDs in the app context, and the enum maps to those IDs. This means we can use the enum to refer to specific event types throughout the system :)

Reply:
Just for context, this is what the table looks like:

<img width="748" alt="Screenshot 2024-05-07 at 16 15 34" src="https://github.com/DFE-Digital/fh-service-directory-shared/assets/3855819/21ec8f44-55f2-4166-9c07-aeecbe4968ca">